Products/ZenModel/migrate/replicateCollectorRedis.py

```diff
@@ -0,0 +1,105 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.1.9")
+
+
+class ReplicateCollectorRedis(Migrate.Step):
+    """
+    Add existence of HBase tables to the Prereqs of OpenTSDB reader.
+    See ZEN-24094
+    """
+
+    version = Migrate.Version(116, 0, 0)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        svcs = ctx.findServices(r".*/localhost/localhost/.*")
```

Comment:
This is only going to catch a vanilla deployment.  We'll need this to run against customers with other collectors that have been added under different hubs.  I *think* that &ast;Collection/&ast; is what we want to search.